packages/core/redux/bitcoin/BitcoinMiddlewareService.js

```diff
@@ -144,7 +151,7 @@ export default class BitcoinMiddlewareService {
     }
 
     // In the testnet we are using blockdozer to get BCC balances
-    if (blockchain === BLOCKCHAIN_BITCOIN_CASH && networkType === 'testnet') {
+    if ([BLOCKCHAIN_BITCOIN_CASH, BLOCKCHAIN_DASH].includes(blockchain) && networkType === 'testnet') {
```

Comment:
Blockdozer service may provide you only with info for Bitcoin, Bitcoin Cash and Litecoin.
Dash does not supported by Blockdozer.